plugin/src/main/java/io/jenkins/plugins/casc/model/Mapping.java

```diff
@@ -9,7 +9,6 @@
 
 public final class Mapping extends HashMap<String, CNode> implements CNode {
 
-    public static final Mapping EMPTY = new Mapping();
```

Comment:
Breaking change?

Reply:
How about keep it. And mark as deprecated.